### PR TITLE
Makes backstabbing a thing

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request
many thanks to @MSieghardt for help 
image

Stabbing people in the back with knifes now deal more harm.

## Changelog
:cl:
/:cl:

### CODE REVIEW

**Review by KathyRyals — CHANGES_REQUESTED** [review]
Don't use a generic name like `backstabing` to indicate a numbered value. 
Instead use something like.. `backstab_damage`.

**Comment by MSieghardt** [comment]
I would recommend adding a check like !user.facing_dir in the IF statement. This is to make sure someone doesn't simply face away from you with forced facing in order to backstab you from the face. Its value is 0 if you aren't facing anything, any other number coresponding with a direction. 

facing_dir isn't an initialized value and receives its first value when used first, but I think that !user.facing_dir will return 1 if the value is NULL too.

**Review by KathyRyals — APPROVED** [review]
Will need testing.

File: code/game/objects/items/weapons/tools/simple_weapons.dm [hunk 1]
```diff
@@ -113,13 +111,24 @@
 	attack_verb = list("slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
 	slot_flags = SLOT_BELT
 	structure_damage_factor = STRUCTURE_DAMAGE_BLADE
+	var/backstab_damage = 5
+
+/obj/item/weapon/tool/knife/apply_hit_effect(atom/target, blocked=FALSE, mob/user)
+	..()
+	if(iscarbon(target) && target.dir == user.dir)
```
Comment (KathyRyals) [inline code/game/objects/items/weapons/tools/simple_weapons.dm:118]: Good catch @MSieghardt .
Implement a check for forced facing so you don't get front-stabbed Dark Souls 1 style.

**Review by KathyRyals — CHANGES_REQUESTED** [review]

**Comment by Trilbyspaceclone** [comment]
I have no idea how to code this

**Pull request closed without merging**